packages/tokens/src/figma-tokens/_docs.json

```diff
@@ -434,6 +434,16 @@
           "value": "The focus event gives the user feedback that the icon-button has been focused by the keyboard interaction.",
           "type": "text"
         }
+      },
+      "variant": {
+        "title": {
+          "value": "Variants",
+          "type": "text"
+        },
+        "description": {
+          "value": "The Icon button can have two color variants, Primary or Neutral Color.",
+          "type": "text"
+        }
       }
```

Comment:
Oh wait a second. I thought you're just showing how to override colors in regard to these two colors. So the plan is to provide those two colors and we have to reflect them in dev as a property?

Then please name it `color` not variant, so we can have the same as in `syn-button`.

Reply:
I changed it quickly (Y)